portfolio/src/main/webapp/script.js

```diff
@@ -128,13 +128,33 @@ function createTaskElement(task) {
     taskElement.remove();
   });
 
+  // var likeButtonElement = document.createElement('button');
+  // likeButtonElement.innerText = 'Like ' + task.likes.toString();
+  // likeButtonElement.style.float = "center";
+  // likeButtonElement.addEventListener('click', () => {
+  //   addLike(task);
+  //   taskElement.remove();
+  // });
+
   taskElement.appendChild(titleElement);
   taskElement.appendChild(nameElement);
   taskElement.appendChild(deleteButtonElement);
   taskElement.appendChild(timeElement);
+  // taskElement.appendChild(likeButtonElement);
   return taskElement;
 }
 
+// function addLike(task) {
```

Comment:
I see that you have added "likes" in the backend. but not ready for frontend yet.

For a pull request, please try to make it in a consistent state.  i.e. each pull request, ideally, should be working and clean.  That way we can confidently rollback if things are broken.

If a change is too big to make it operational, please try to break up the pieces in a more self-contained format. For example, the "likes" feature is not working in this pull request, you can leave it out. So the reviewer can focus on the code that actually works.